https://patchwork.ozlabs.org/patch/706460/

From f4916521c6971dba1626d009db8ae412d1f494b2 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.xyz>
Date: Fri, 16 Dec 2016 22:35:02 +0800
Subject: [PATCH 2/2] sunxi: add orangepi zero defconfig

Orange Pi Zero is a board designed by Xunlong. It has an Allwinner H2+
SoC (similar to H3, which shares the same SoC ID), 256MB/512MB RAM,
Allwinner XR819 SDIO Wi-Fi, a MicroUSB port which is used to power the
board (also capable of OTG), a USB Type-A socket and a MicroSD slot.

Signed-off-by: Icenowy Zheng <icenowy@aosc.xyz>
Reviewed-by: Jagan Teki <jagan@openedev.com>
---
 board/sunxi/MAINTAINERS         |  5 +++++
 configs/orangepi_zero_defconfig | 15 +++++++++++++++
 2 files changed, 20 insertions(+)
 create mode 100644 configs/orangepi_zero_defconfig

Index: u-boot/board/sunxi/MAINTAINERS
===================================================================
--- u-boot.orig/board/sunxi/MAINTAINERS
+++ u-boot/board/sunxi/MAINTAINERS
@@ -237,6 +237,11 @@ M:	FUKAUMI Naoki <naobsd@gmail.com>
 S:	Maintained
 F:	configs/Nintendo_NES_Classic_Edition_defconfig
 
+ORANGEPI ZERO BOARD
+M:	Icenowy Zheng <icenowy@aosc.xyz>
+S:	Maintained
+F:	configs/orangepi_zero_defconfig
+
 R16 EVB PARROT BOARD
 M:	Quentin Schulz <quentin.schulz@free-electrons.com>
 S:	Maintained
Index: u-boot/configs/orangepi_zero_defconfig
===================================================================
--- /dev/null
+++ u-boot/configs/orangepi_zero_defconfig
@@ -0,0 +1,15 @@
+CONFIG_ARM=y
+CONFIG_ARCH_SUNXI=y
+CONFIG_MACH_SUN8I_H3=y
+CONFIG_DRAM_CLK=672
+CONFIG_DRAM_ZQ=3881979
+CONFIG_DRAM_ODT_EN=y
+CONFIG_DEFAULT_DEVICE_TREE="sun8i-h2-plus-orangepi-zero"
+# CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
+CONFIG_CONSOLE_MUX=y
+CONFIG_SPL=y
+# CONFIG_CMD_IMLS is not set
+# CONFIG_CMD_FLASH is not set
+# CONFIG_CMD_FPGA is not set
+CONFIG_SUN8I_EMAC=y
+CONFIG_USB_EHCI_HCD=y
